<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yoobit翻译——From:贾镇宇</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="translate_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>033-上首方糖体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style: outset;
background-color:rgb(194, 255, 253);
border-radius: 10px;  border: 2px groove gray;
}</string>
    </property>
    <property name="text">
     <string>翻译</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_text">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>261</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>033-上首方糖体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{
background-color:rgb(247, 220, 255);
	border-radius: 10px;  border: 2px groove gray;
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sel_language">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>033-上首方糖体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(255, 251, 147);
        border-radius: 10px;  border: 2px groove gray;
        border-style: outset;

}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_word_book">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>033-上首方糖体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style: outset;
background-color:rgb(147, 203, 255);
border-radius: 10px;  border: 2px groove gray;
}</string>
    </property>
    <property name="text">
     <string>加入单词本</string>
    </property>
   </widget>
   <widget class="QPushButton" name="watch_word_book">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>033-上首方糖体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style: outset;
background-color:rgb(134, 255, 215);
border-radius: 10px;  border: 2px groove gray;
}</string>
    </property>
    <property name="text">
     <string>查看单词本</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>033-上首方糖体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
	background-color: rgb(255, 176, 160);
border-radius: 10px;  border: 2px groove gray;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>--&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
